client.go

```diff
@@ -46,6 +47,8 @@ type Client struct {
 
 type ClientOption func(c *Client)
 
+var ClientTimeout = 1 * time.Minute
```

Comment:
This shouldn't be exported. If it's intended to be configurable then it should be passed in as a `ClientOption`.

Reply:
Will pull into a separate PR.